github_copilot_seats/triggers.py

```diff
@@ -0,0 +1,32 @@
+from autokitteh.slack import slack_client
+from seats import prune_idle_seats, find_idle_seats
+
+s = slack_client("slack_conn")
+
+
+def on_schedule() -> None:
+    "Entry point for the workflow"
+    print(prune_idle_seats())
+
+
+def on_slack_slash_command(event) -> None:
+    "Entry point for the workflow"
+    cmd = event.data.text
+    cid = event.data.channel_id
+
+    if cmd not in {"prune-idle-copilot-seats", "find-idle-copilot-seats"}:
+        s.chat_postMessage(channel=cid, text="Unrecognized command")
```

Comment:
i wonder if we're using multiple automations on the same bot, this would give you multiple "unrecognized command" if implemented this way in multiple workflows. @daabr wdyt?

Reply:
I think you're right about that.

Reply:
Yes. Ideally, people will use custom OAuth after their initial experiments.